include/ttmlir/Dialect/TTKernel/IR/TTKernelOpsTypes.td

```diff
@@ -31,8 +31,8 @@ def TTKernel_CB : TTKernel_Type<"CB", "cb"> {
         static CBType get(::mlir::MLIRContext *context,
                               uint64_t address,
                               uint64_t port,
-                              MemRefType memref,
-                              uint64_t numBuffers = 1) {
+                              MemRefType memref) {
+          uint64_t numBuffers = 1;
```

Comment:
This is unrelated, but an important bug fix.  I can break it out if folks think I should